block_iagora.php

```diff
@@ -61,4 +69,27 @@ public function get_content() {
     public function applicable_formats() {
         return ['all' => true];
     }
+     
+    /**  
+     * Allow the block to be configured.
+     * 
+    * @return boolean
```

Comment:
```suggestion
    * @return bool
```
Moodle uses short type names (bool instead of boolean, int instead of integer). See [Style guide](https://moodledev.io/general/development/policies/codingstyle#types).

Reply:
okay